Sources/XCTest/Public/Asynchronous/XCTestCase+NotificationExpectation.swift

```diff
@@ -45,8 +45,7 @@ public extension XCTestCase {
             .default
             .addObserver(forName: Notification.Name(rawValue: notificationName),
                          object: objectToObserve,
-                         queue: nil,
-                         usingBlock: {
+                         queue: nil) {
```

Comment:
Added this change in a separate PR, see apple/swift-corelibs-xctest#177. I rebased this PR on PR with the syntax cleanup. In other case iOS build fails. I'll rebase on master as soon as the syntax cleanup approved.